ODIN_II/CMakeLists.txt

```diff
@@ -2,6 +2,12 @@ cmake_minimum_required(VERSION 3.9)
 
 project("ODIN_II")
 
+#
+# ODIN_II Build Options
+#
+set(VTR_COMPILE_OPTIONS "none" CACHE STRING "Specify whether ODIN_II uses strict compiling options, e.g. -Werror")
+set(CACHE VTR_COMPILE_OPTIONS PROPERTY STRING strict none)
```

Comment:
It looks like VTR_COMPILE_OPTIONS is defined both here (ODIN_II/CMakeLists.txt) and in vpr (vpr/CMakeLists.txt).

I'm not sure how CMake will handle this. Could we move it so it is defined in the root CMakeLists.txt file? That way there will only be one definition.

Reply:
Sure, moved it in root CMakeLists.txt